views/management/forms/instance-policy.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="form-group">
+	<label>Titel</label>
+	<input class="form-control" name="consentTitle" type="text" data-force-value="true" required>
+</div>
+<div class="form-group">
+	<label>Beschreibung</label>
+	<textarea class="form-control" name="consentText" required></textarea>
+</div>
+<div class="form-group">
+	<label>Datei hochladen (nur PDF, max 4MB)</label><br/>
+	<input type="hidden" id="policy-file-data" name="consentData" value="{{consentData}}">
+	<input type="file" id="policy-input" accept="application/pdf" required>
+	<label for="policy-input" class="btn btn-secondary">Datei auswählen</label>
+</div>
+<div class="form-group">
+	<div class="policy-file-upload">
+		<img id="policy-file-logo" src="/images/thumbs/pdfs.png" width="25px">

```

Comment:
meaningful alt or role presentation. Can we also have rems instead of pxs?

Reply:
this has been copied from the legacy client. @luizzaly already did a review of how it looks, maybe we should not change it?

Reply:
okay, double check with Luisa but imho I think it should be added ;-)

Reply:
@luizzaly can you confirm what should I do with this 25px? Change it or not? I have checked how it looks for smaller and bigger devices and for me it is ok.